### ISSUE TITLE
[FEATURE] Set Default Home Page & Enable Cyclic Swiping

### ISSUE THREAD

**Issue by 3Domse3** [Deletescape-Media/Lawnchair#4733]
### **Set Default Home Page:**
Add an option to select the default or "home" page on the home screen that appears when the home button is pressed. This feature would allow users to designate any page on their home screen as the "main" page to which they return after pressing the home button.

### **Enable Cyclic Swiping:**
Implement cyclic swiping on the home screen, allowing users to continuously swipe in either direction (left or right) through their home screen pages without hitting an endpoint. For example, swiping right from the last page would take the user back to the first page, and vice versa.

### Set Default Home Page:

**Personalization:** 
Users often have different priorities for the content on their home screens. Some may prefer to have their widgets, while others may prioritize apps. Allowing users to set a specific page as their default will enable them to customize their experience further, making the launcher more aligned with their preferences and workflows.

**Efficiency:** 
Users can jump directly to their most frequently used page without needing to swipe through other pages, enhancing navigation speed and overall user experience.

### Enable Cyclic Swiping:

**Ease of Navigation:** 
Cyclic swiping eliminates the need to reverse direction when navigating through multiple home screen pages. This fluidity simplifies the user experience, especially for those with many home screen pages.

**Consistency:**
Many modern launchers and applications use cyclic navigation, so implementing this feature would bring the launcher in line with user expectations and contemporary design standards.

**Ergonomics:**
Continuous swiping reduces the need for repetitive swipes in a single direction, making the user interaction smoother and more comfortable, especially for users with accessibility needs or on larger screens.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

For example OneUI from Samsung

**Comment by ckinwai** [comment]
Totally agree this feature is very nice to have!

**Comment by tm-frs** [comment]
I agree as well, especially not being able to set a default panel is really annoying

**Comment by chofnar** [comment]
Would very much like this to be added, too. It's overwhelming when you have 5 or more screens.

**Comment by Exaper** [comment]
Same here. Ready to jump off Nova launcher to this one if only there was a way to set default home page.

**Comment by Exaper** [comment]
I've opened PR that adds this functionality: https://github.com/LawnchairLauncher/lawnchair/pull/6395

**Comment by validcube** [comment]
This issue have been fixed by:
* #6395 (will be included in the next release of Lawnchair 16)
* #5807 (will be included in the next release of Lawnchair 15)
